how-to/how-to-use-k8s-with-cri-containerd-and-kata.md

```diff
@@ -23,13 +23,13 @@ to launch Kata Containers. For the previous version of Kata Containers, the Pods
 
 ## Requirements
 
-- Kubernetes, kubelet, kubeadm
+- Kubernetes, Kubelet, `kubeadm`
```

Comment:
well, technically `kubelet` is a command, it just happens to be run by the init/system and not normally by hand... but, there is a binary called `kubelet` (unlike Kubernetes for instance).

Reply:
So, should we add a custom "kubelet" word to our dictionary, or use backticks folks?

Reply:
i think add it to dictionary.